src/views/udns/list/components/utils.ts

```diff
@@ -53,3 +59,11 @@ export const getDefaultUDN = (isClusterUDN: boolean, namespace: string): UDNForm
         namespace === ALL_NAMESPACES_KEY ? DEFAULT_NAMESPACE : namespace,
       );
 };
+
+export const isUDNValid = (udn: UDNForm): boolean => {
+  const clusterUDNConnected =
+    (udn?.spec?.namespaceSelector?.matchExpressions || []).length > 0 ||
```

Comment:
same comments like other pr, probably need rebase after the first will be merged?